build-system/tasks/check-links.js

```diff
@@ -141,6 +141,9 @@ function filterWhitelistedLinks(markdown) {
 
   // Links inside a <code> block (illustrative, and not always valid)
   filteredMarkdown = filteredMarkdown.replace(/<code>(.*?)<\/code>/g, '');
+  
+  // Links inside a <pre> block (illustrative, and not always valid)
+  filteredMarkdown = filteredMarkdown.replace(/<pre>(.*?)<\/pre>/g, '');
```

Comment:
Hmm. This doesn't work across multiple lines as written. Let me get back to you with a version of the regex that works across lines.

Reply:
The `s` flag is super new. We can use `[^]` in place of `.` and get the same effect.

Reply:
Done in #14995.